src/applications/gi/reducers/calculator.js

```diff
@@ -72,6 +85,36 @@ export default function (state = INITIAL_STATE, action) {
 
     case FETCH_PROFILE_SUCCEEDED: {
       const camelPayload = camelCaseKeysRecursive(action.payload);
+      const yellowRibbonPrograms = [
+        {
+          divisionProfessionalSchool: 'division1',
+          numberOfStudents: 5,
+          degreeLevel: 'graduate',
+          contributionAmount: 5000
+        },
+        {
+          divisionProfessionalSchool: 'division2',
+          numberOfStudents: 3000,
+          degreeLevel: 'undergraduate',
+          contributionAmount: 5
+        },
+        {
+          divisionProfessionalSchool: 'division3',
+          numberOfStudents: 20,
+          degreeLevel: 'undergraduate',
+          contributionAmount: 25
+        }
+      ];
+
+      const yellowRibbonDegreeLevelOptions = yellowRibbonPrograms.length > 0 ?
```

Comment:
Decided to denormalize this rather than deal with the entire `yellowRibbonPrograms` array.